.circleci/config.yml

```diff
@@ -259,13 +256,10 @@ jobs:
           name: upload docker image
           no_output_timeout: "1h"
           command: |
-            set +x
-            export AWS_ACCESS_KEY_ID=${ECR_AWS_ACCESS_KEY}
-            export AWS_SECRET_ACCESS_KEY=${ECR_AWS_SECRET_ACCESS_KEY}
-            eval $(aws ecr get-login --region us-east-1 --no-include-email)
             set -x
-            docker tag ${image_name}:${CIRCLE_WORKFLOW_ID} 308535385114.dkr.ecr.us-east-1.amazonaws.com/${image_name}:${CIRCLE_WORKFLOW_ID}
-            docker push 308535385114.dkr.ecr.us-east-1.amazonaws.com/${image_name}:${CIRCLE_WORKFLOW_ID}
+            docker tag ${image_name}:${CIRCLE_WORKFLOW_ID} ${image_name}:latest
```

Comment:
Oh we really shouldn't be using docker images here as intermediary storage.

I'd prefer it if we just removed this and used circleci workspaces.

Here's some documentation on how to do that: https://circleci.com/blog/deep-diving-into-circleci-workspaces/

Reply:
@seemethere or can we just do build and push in one step?

Reply:
@seemethere I merged the steps so that there is no need to pass data between steps.

Reply:
Oh I see, this build would still fail though since you don't actually have credentials in order to push these images, let me see if I can resolve that for you.